Dart/resources/META-INF/plugin.xml

```diff
@@ -289,5 +290,22 @@
       <separator/>
     </group>
 
+    <action id="DartCallHierarchy.BaseOnThisFunction" text="Base on This Function"
+            class="com.jetbrains.lang.dart.ide.hierarchy.call.DartCallHierarchyBrowser$BaseOnThisFunctionAction"
```

Comment:
OK. I also changed the menu text from "Function" to "Component" to hopefully be more indicative that it works for getter, setters, and constructors, too.